### PR TITLE
update tutorials utilizing icepyx

### PR DESCRIPTION
**Pull request by JessicaS11**
icepyx [released v1.0.0](https://medium.com/@icepyx.dev/the-icepyx-developers-are-thrilled-to-announce-the-release-of-icepyx-v1-0-0-678dfdc05c31) in January 2024. This PR updates the CryoCloud tutorials that utilize icepyx to use the latest functionality.

### CODE REVIEW

File: book/tutorials/IS2_ATL15_surface_height_anomalies/IS2_ATL15_surface_height_anomalies.ipynb
Comment (JessicaS11) [inline book/tutorials/IS2_ATL15_surface_height_anomalies/IS2_ATL15_surface_height_anomalies.ipynb:1]: Note: I know I still need to run this one for outputs

Comment (tsnow03) [reply book/tutorials/IS2_ATL15_surface_height_anomalies/IS2_ATL15_surface_height_anomalies.ipynb:1]: Do you want me to wait to approve?

Comment (JessicaS11) [reply book/tutorials/IS2_ATL15_surface_height_anomalies/IS2_ATL15_surface_height_anomalies.ipynb:1]: Yes, please wait - the most recent version of ATL15 has some [metadata issues](https://forum.earthdata.nasa.gov/viewtopic.php?t=5154) that I'd like to get temporary fixes into icepyx for (and then into the hub image) so users don't need a dev install to run that tutorial.

Is there a way to view the NetlifyPreview? I'd like to see if any of the image-rendering issues we had at the AGU workshop are resolved (cc @asteiker).

Comment (JessicaS11) [reply book/tutorials/IS2_ATL15_surface_height_anomalies/IS2_ATL15_surface_height_anomalies.ipynb:1]: [Waiting on review](https://github.com/icesat2py/icepyx/pull/516) to get the fix into icepyx and push a new release so this example will work out of the box on CryoCloud (i.e. without a dev install of icepyx).

**Pull request merged**